9.0/wheezy/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM buildpack-deps:wheezy
```

Comment:
can we remove this? Ref #466 @chorrell

Reply:
Or do we just want to remove it all at once?

Reply:
I believe that we can remove it later?

Reply:
Just seems odd to add something that is superseded. Not really an issue as much as unneeded. No biggie though, feel free to merge :)